activerecord_json_validator.gemspec

```diff
@@ -18,15 +18,15 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  spec.add_development_dependency 'bundler', '~> 1.3'
+  spec.add_development_dependency 'bundler', '~> 1.12'
   spec.add_development_dependency 'rake'
-  spec.add_development_dependency 'rspec', '~> 3.1'
+  spec.add_development_dependency 'rspec', '~> 3.5'
   spec.add_development_dependency 'pg'
   spec.add_development_dependency 'mysql2'
-  spec.add_development_dependency 'activesupport', '>= 4.1.0', '< 5'
+  spec.add_development_dependency 'activesupport', '>= 4.1.0', '< 6'
```

Comment:
I think we could match `activesupport` version requirement with `activerecord`’s.